samples/mpu/mpu_test/README.rst

```diff
@@ -46,6 +46,15 @@ single thread: :file:`prj_single.conf`:
 
    $ make BOARD=v2m_beetle CONF_FILE=prj_single.conf run
 
+To build the version that allows flash write, use the supplied configuration
+file: :file:`prj_allow_flash_write_<x>.conf`:
```

Comment:
before the trailing colon, add the explanation about `<x>` instead of after the code block:

     file: :file:`prj_allow_flash_write_<x>.conf`, replacing `<x>` with either `mcux` or `stm32`: